### PR TITLE
Model

### PR DESCRIPTION
**Pull request by thanhpv958**

### CODE REVIEW

File: app/ImageUpload.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class ImageUpload extends Model
+{
+    protected $table='image_uploads';
```
Comment (BunnyPi04) [inline app/ImageUpload.php]: convention

File: app/Room.php [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Room extends Model
+{
+    protected $table='rooms';
```
Comment (BunnyPi04) [inline app/Room.php]: convention

File: app/Theater.php [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Theater extends Model
+{
+    protected $table='theaters';
```
Comment (BunnyPi04) [inline app/Theater.php]: convention

File: app/Film.php [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    protected $table = 'films';
+
+    public function categories()
+    {
+        return $this->belongsToMany('App\Category');
+    }
+
+    public function calendars()
+    {
+        return $this->hasMany('App\Calendar');
```
Comment (trinhductoan) [inline app/Film.php]: `return $this->hasMany(Calendar::class);`

**Pull request merged**